spec/03-plugins/37-opentelemetry/01-otlp_spec.lua

```diff
@@ -0,0 +1,114 @@
+require "spec.helpers"
+require "kong.plugins.opentelemetry.proto"
+local otlp = require "kong.plugins.opentelemetry.otlp"
+local utils = require "kong.tools.utils"
+local pb = require "pb"
+
+local fmt = string.format
+local rand_bytes = utils.get_rand_bytes
+local time_ns = utils.time_ns
+local insert = table.insert
+local tracer = kong.tracing.new("test")
+
+-- test assert only
+pb.option("enum_as_value")
```

Comment:
this may affect other tests, let's move this to `setup` and revert it back in `teardown`

Reply:
Ops, it's global and this could be affecting wRPC tests. (or other things using lua-protobuf)